tests/python/04_swizzle/main.py

```diff
@@ -1,41 +1,37 @@
 #!/usr/bin/python3
 
 import sys
+import traceback
 import uuid
-from xrt_binding import * # found in PYTHONPATH
+import numpy
+# found in PYTHONPATH
+import pyxrt
+
 sys.path.append('../') # utils_binding.py
 from utils_binding import *
 
-XVECTORSWIZZLE_CONTROL_ADDR_AP_CTRL = 0x00
-XVECTORSWIZZLE_CONTROL_ADDR_GIE = 0x04
-XVECTORSWIZZLE_CONTROL_ADDR_IER = 0x08
-XVECTORSWIZZLE_CONTROL_ADDR_ISR = 0x0c
-XVECTORSWIZZLE_CONTROL_ADDR_GROUP_ID_X_DATA = 0x10
-XVECTORSWIZZLE_CONTROL_BITS_GROUP_ID_X_DATA = 32
-XVECTORSWIZZLE_CONTROL_ADDR_GROUP_ID_Y_DATA = 0x18
-XVECTORSWIZZLE_CONTROL_BITS_GROUP_ID_Y_DATA = 32
-XVECTORSWIZZLE_CONTROL_ADDR_GROUP_ID_Z_DATA = 0x20
-XVECTORSWIZZLE_CONTROL_BITS_GROUP_ID_Z_DATA = 32
-XVECTORSWIZZLE_CONTROL_ADDR_GLOBAL_OFFSET_X_DATA = 0x28
-XVECTORSWIZZLE_CONTROL_BITS_GLOBAL_OFFSET_X_DATA = 32
-XVECTORSWIZZLE_CONTROL_ADDR_GLOBAL_OFFSET_Y_DATA = 0x30
-XVECTORSWIZZLE_CONTROL_BITS_GLOBAL_OFFSET_Y_DATA = 32
-XVECTORSWIZZLE_CONTROL_ADDR_GLOBAL_OFFSET_Z_DATA = 0x38
-XVECTORSWIZZLE_CONTROL_BITS_GLOBAL_OFFSET_Z_DATA = 32
-XVECTORSWIZZLE_CONTROL_ADDR_A_DATA = 0x40
-
 def runKernel(opt):
+    result = 0
+    d = pyxrt.device(opt.index)
+    uuid = d.load_xclbin(opt.bitstreamFile)
+    # Instantiate vectorswizzle
+    swizzle = pyxrt.kernel(d, uuid.get(), "vectorswizzle")
```

Comment:
Not necessary to call get() on the uuid.

Reply:
Done